environment.yml

```diff
@@ -1,31 +1,33 @@
 name: dask-tutorial
 channels:
-  - defaults
   - conda-forge
 dependencies:
-  - python=3.6
-  - numpy
+  - python=3.7
+  - numpy=1.16.4
+  - nomkl
   - h5py
-  - scipy
+  - scipy=1.3.0
   - toolz
-  - bokeh
-  - dask
-  - distributed
+  - bokeh=1.2.0
+  - dask=1.2.2
+  - distributed=1.28.1
   - notebook
   - matplotlib
   - Pillow
-  - pandas
+  - pandas=0.24.2
   - pandas-datareader
   - pytables
-  - scikit-learn
-  - scikit-image
+  - scikit-learn=0.21.1
+  - scikit-image=0.15.0
   - snakeviz
   - ujson
   - graphviz
   - pip
-  - s3fs
-  - fastparquet
-  - dask-ml
+  - s3fs=0.2.1
+  - fastparquet=0.3.0
+  - dask-ml=0.12.0
+  - ipywidgets
   - pip:
-    - graphviz 
+    - graphviz
     - cachey
```

Comment:
This has [since been added to conda-forge]( https://github.com/conda-forge/cachey-feedstock ). Would it make sense to use the Conda package for it instead?